homeassistant/helpers/entityfilter.py

```diff
@@ -133,15 +144,15 @@ def entity_included(domain: str, entity_id: str) -> bool:
         return (
             entity_id in include_e
             or domain in include_d
-            or any(glob_re.match(entity_id) for glob_re in include_eg)
+            or _test_against_patterns(include_eg, entity_id)
```

Comment:
```suggestion
            or (include_eg and _test_against_patterns(include_eg, entity_id))
```

If there isn't a list to match against, we can avoid having to pay the cost of calling `_test_against_patterns` so filters without glob matchers have minimal performance impact. (more places need this as well)

Reply:
One of the reasons we are building lots of functions here is we expect them to be called in tight loops so anything we can do to not have to jump to another function in the loop will keep this performant

Reply:
Ok makes sense to me.  I'll add that the and conditional in front of all of them